cloudify_agent/installer/__init__.py

```diff
@@ -122,14 +122,19 @@ def _create_agent_env(self):
                                  broker_config.broker_password)
         broker_vhost = tenant.get('rabbitmq_vhost',
                                   broker_config.broker_vhost)
-
+        rest_host = manager_ip if manager_ip else \
```

Comment:
Better change to:
`rest_host = manager_ip or self.cloudify_agent['rest_host']`
Then it will fit in one line